### PR TITLE
DEVDOCS-3079: Re-add Firebase setup instructions

### PR DESCRIPTION
**Pull request by ektravel**
# [DEVDOCS-3079](https://jira.bigcommerce.com/browse/DEVDOCS-3079)

## What changed?
* Added Firebase setup instructions
* Added OAuth scope to quick start

### CODE REVIEW

**Review by traci707 — COMMENTED** [review]
I went through the steps until I got to the _Create a Firebase config_ and _Create a Cloud Firestore database_ sections. The option to select the **web** platform was not available. Also, I could not find **Firestore Database** in the left pane.
I made updates reflecting what I see on my end. It does seem that I don't have all permissions as I was not able to create a project.  I would get an error each time and I could not access the other projects. I could only see the demo project.

**Comment by ektravel** [comment]
@traci707 I've added some clarifications to the Firebase instructions.

File: docs/api-docs/apps/tutorials/sample-app-next/step-3-integrate-api.md [hunk 1]
```diff
@@ -317,29 +317,25 @@ For MySQL configuration instructions, see [Set up MySQL database](#set-up-mysql-
...
+5. Click **Create project**.
 
 ### Create a Firebase config
 
 1. In your Firebase project console, click on the **settings icon** that resembles a gear in the top left corner.
 
 2. Select **Project settings** from the dropdown menu.
 
-3. Under the **General** tab, scroll down to **Your apps** and select the **web** platform.
+3. Under the **General** tab, scroll down to **Your apps** and click on the **code icon** to select the web platform.
```
Comment (traci707) [inline docs/api-docs/apps/tutorials/sample-app-next/step-3-integrate-api.md:330]: 3. Under the **General** tab, scroll down to **Your apps** and click on the **code icon** (</>) to select the web platform.

**Review by traci707 — COMMENTED** [review]
looks good, I found just one thing.

**Comment by ektravel** [comment]
@uwikaiddi this PR is ready for final review.

**Review by uwikaiddi — APPROVED** [review]

**Pull request merged**